src/discordcr/mappings/vws.cr

```diff
@@ -4,104 +4,94 @@ module Discord
   # :nodoc:
   module VWS
     struct IdentifyPacket
+      include JSON::Serializable
       def initialize(server_id, user_id, session_id, token)
```

Comment:
i think its a good practice to separate the `include JSON::Serializable` with one newline

e.g.

```cr
class Klass
  include JSON::Serializable

  property username : String?
end